website_apps_store/controllers/main.py

```diff
@@ -19,10 +19,16 @@
 
 class WebsiteSaleCustom(WebsiteSale):
 
-    def _get_search_domain(self, search, category, attrib_values):
+    def _get_search_domain(self, search, category, attrib_values,
+                           search_all_terms=''):
```

Comment:
What do you think about search_option instead?
Now we split the words but maybe later we want to add additional search options.
Would it be more flexible?